### PR TITLE
File Handling for Codegen

### PR DESCRIPTION
**Pull request by AmmarD212**
This pull request adds a core piece to the helper.js file in lib mainly getContentType and buildRequestFileObject. The tests are included in test/unit. The NLC service file was reverted to the original one from master to shift focus on the changes to the core sdk piece for this pull request.

##### Checklist

- [x] `npm test` passes (tip: `npm run autofix` can correct most style issues)
- [x] tests are included
- [x] documentation is changed or added
- [ ] link to public docs when adding new a service or new features for an existing service

##### New version_date Checklist

- [ ] A new constant is avaliable with the version_date - [example](https://github.com/watson-developer-cloud/node-sdk/blob/d1418ac2f9774194aaff0c8bd80f0d3722beef72/conversation/v1.js#L77)
- [ ] The new constant has a comment that summarizes the changes and/or links to relevant doc pages
- [ ] Any older version_date constants remain intact
- [ ] The error message thrown if the service is created without a version_date indicates the new version_date constant
- [ ] The example in the README includes the new version_date constant
- [ ] Any relevant code in the examples/ folder has been updated to use the new version_date constant
- [ ] Most tests are updated to the new version_date
- [ ] 1-2 new tests are added that use the old version_date (optional, but preferred)

### CODE REVIEW

File: lib/helper.js [hunk 1]
```diff
@@ -71,5 +97,51 @@ module.exports = {
       }
     }
     return null;
+  },
+  /**
+   * this function builds a `form-data` object for each file parameter
+   * @param {Object} fileParams an object with fields data and contentType
```
Comment (germanattanasio) [inline lib/helper.js]: jsdoc can be added for each field in `fileParams`. See PersonalityInsightsV3

Comment (AmmarD212) [reply lib/helper.js]: Will do.

**Comment by glennrfisher** [comment]
Looks like this was merged successfully. Can the `file-handling-codegen` branch be deleted?

**Comment by AmmarD212** [comment]
Yes it can @glennrfisher  👍

**Pull request merged**